app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DashboardController < ApplicationController
```

Comment:
Are we eventually going to have fixity and other things on a single dashboard?  If so, I'm fine with `DashboardController`.  But if this is only going to be for fixity, it might be better to call it `FixityController` instead.

Reply:
Yeah, my thinking was that it could be built on as we found more things we needed to put on a dashboard, with some sort of tabbed UI. Happy to change it, though, if you don't think there will be such use cases, or if you just prefer to wait until we actually implement them. I just thought it would be a helpful flag for remembering to do so if/when the time comes, down the line.

Reply:
That makes sense.  Though it makes me think the recently-created ReportsController should be folded into this, too.

Reply:
I considered that pretty seriously. In the end i sort of thought reports and dashboards are different things. but maybe that doesn't matter because they have similar use cased scenarios? superuser-type functionality, etc. Shall I fold the report in and call it all "dashboard"?

Reply:
👍 Sounds good to me.